.github/actions/docker-images/action.yml

```diff
@@ -0,0 +1,112 @@
+name: 'Docker Images'
+description: 'Build docker images'
+inputs:
+  build-docker-images:
+    description: "Build docker images"
+    required: false
+    default: 'true'
+
+  container-image-suffix:
+    description: 'Container image suffix'
+    required: false
+    default: ${{ github.run_id }}
+  elastic-agent-docker-image:
+    description: 'Elastic-Agent docker image'
+    required: false
+    default: "704479110758.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent"
+  elastic-agent-docker-image-tag:
+    description: 'Elastic-Agent docker image tag'
+    required: false
+    default: "8.14.0-SNAPSHOT"
```

Comment:
Do we want this default? We will pass 8.14 in one or two months and forget this

Reply:
I am not sure I understood if you suggest using a different tag or something entirely different. 

This is how it works currently: We need a base image in which we build the agent and patched cloudbeat to test. 

We could explore updating those values automatically in a separate job, perhaps, but I believe this is out of the scope of this PR, right?

Reply:
Can be outside of the scope.

I'm just not a fan of a default with a tag that will be outdated very soon

Reply:
> I am not sure I understood if you suggest using a different tag or something entirely different.
> 
> This is how it works currently: We need a base image in which we build the agent and patched cloudbeat to test.
> 
> We could explore updating those values automatically in a separate job, perhaps, but I believe this is out of the scope of this PR, wdyt?

I think APM implemented an action for something like that